src/domain/tradingview/utils.ts

```diff
@@ -0,0 +1,13 @@
+export const supportedResolutions = { 5: "5m", 15: "15m", 60: "1h", 240: "4h", "1D": "1d" };
+export const LAST_BAR_REFRESH_INTERVAL = 15000; // 15 seconds
+
+export function getPeriodFromResolutions(value, object = supportedResolutions) {
+  return Object.keys(object).find((key) => object[key] === value);
+}
+
+export function formatTimeInBar(bar) {
```

Comment:
It is not clear what the format of the bar is, it seems to me that we should add the types here

Reply:
Type added